File: Go/uri_1048.go
package main

import "fmt"

func main() {
	var salary, new_salary, readjust float64
	r := [5]float64{0.04, 0.07, 0.10, 0.12, 0.15}
	var sel int
	fmt.Scan(&salary)
	if salary <= 400.00 {
		sel = 4
		readjust = salary * r[sel]

	} else if salary >= 400.01 && salary <= 800.00 {
		sel = 3
		readjust = salary * r[sel]
	} else if salary >= 800.01 && salary <= 1200.00 {
		sel = 2
		readjust = salary * r[sel]
	} else if salary >= 1200.01 && salary <= 2000.00 {
		sel = 1
		readjust = salary * r[sel]
	} else {
		sel = 0
		readjust = salary * r[sel]
	}
	new_salary = salary + readjust
	fmt.Printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %.0f %%\n", new_salary, readjust, r[sel]*100)
}

File: Python/uri_1048.py
n = float(input())
if 0 <= n <= 400.00:
    t = 0.15
elif 400.01 <= n <= 800.00:
    t = 0.12
elif 800.01 <= n <= 1200.00:
    t = 0.10
elif 1200.01 <= n <= 2000.00:
    t = 0.07
else:
    t = 0.04
print('Novo salario: %0.2f' % (n + (n * t)))
print('Reajuste ganho: %0.2f' % (abs(n - (n + (n * t)))))
print('Em percentual: %0.0f %%' % (t * 100))